content/en/docs/faq/_index.md

```diff
@@ -20,19 +20,50 @@ face:
 
 This is a feature in cert-manager starting in `v0.16` using the kubectl plugin. More information can be found on [the renew command's page](../usage/kubectl-plugin/#renew)
 
-### How can I see all the historic events related to a certificate object ?
+### Why isn't my root certificate in my issued Secret's `tls.crt`?
+
+Occasionally, people work with systems which have made a flawed choice regarding TLS chains. The [TLS spec](https://datatracker.ietf.org/doc/html/rfc5246#section-7.4.2)
+has the following section for the "Server Certificate" section of the TLS handshake:
+
+> This is a sequence (chain) of certificates.  The sender's
+> certificate MUST come first in the list.  Each following
+> certificate MUST directly certify the one preceding it.  Because
+> certificate validation requires that root keys be distributed
+> independently, the self-signed certificate that specifies the root
+> certificate authority MAY be omitted from the chain, under the
+> assumption that the remote end must already possess it in order to
+> validate it in any case.
+
+In a standard, secure and correctly configured TLS environment, adding a root certificate to the chain is almost entirely _pure waste_.
+
+There are two ways that a certificate can be trusted:
+
+- explicitly, by including it in a trust store.
+- through a signature, by following the certificate's chain back up to an explicitly trusted certificate.
+
+Crucially, root certificates are by definition self-signed and they cannot be validated through a signature.
+
+As such, if we have a client trying to validate the certificate chain sent by the server, the client must already have the
+root before the connection is started. If the client already has the root, there was no point in it being sent by the server!
```

Comment:
Should we also say something about hypothetical client mounting tls.crt as a client-certificate? Can we link to a corresponding section of the spec that says whether the client MAY / SHOULD / MUST / or MUST NOT send the full chain to the server?

Reply:
The [spec](https://datatracker.ietf.org/doc/html/rfc5246#section-7.4.6) points at the same justification for server certificates. I'll add a line explaining this.